lib/matplotlib/tests/test_axes.py

```diff
@@ -4964,6 +4964,26 @@ def test_loglog():
     ax.tick_params(length=15, width=2, which='minor')
 
 
+@image_comparison(baseline_images=["test_loglog_nonposx_clip"],
```

Comment:
save an image by using subplots? :-)

Reply:
Umm yeah that’s fair.  I can actually just plot twice on the same axis.  Or four times to get all four cases.   Stand by